assets/js/script.js

```diff
@@ -205,7 +205,7 @@ function renderGame(game, homeAway) {
   }
   gameEl.append(gameStatusEl);
   // gameEl.append(saveBtnEl);
-  $("#upcoming-games").append(gameEl);
+  $("#schedule").append(gameEl);
```

Comment:
I made this change earlier and that's why it worked properly before I made my previous PR.
But lost this change while resolving merge conflicts.